repository-data/site/src/main/resources/hcm-config/hst/configurations.yaml

```diff
@@ -10,7 +10,7 @@ definitions:
         jcr:primaryType: hst:configuration
       /intranet:
         hst:inheritsfrom:
-        - ../base
+        - ../common
```

Comment:
Have you tested for regressions?

Reply:
Anything in intranet will override the main site config, this will just save us copying over pages/templates for the website doctypes. It can only break things that were previously broken anyway

Reply:
fair enough